modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -35,15 +35,12 @@ _Introduced in DataWeave 2.5.0. Supported by Mule 4.5.0 and later._
 
 DataWeave can read and write doctype declarations in XML files.
 
-To read a doctype declarations, set the xref:dataweave-system-properties.adoc[system property] `com.mulesoft.dw.xml_reader.parseDtd`.
-
-During the reading phase, DataWeave parses the doctype declarations and stores the content as metadata of the root element. The `DocType` value is stored in the `docType` variable. You can use the metadata selector (`^`) to extract the value of that variable with an expression like this one:
+During the reading phase, DataWeave parses the doctype declarations and stores the content as metadata of the root element. The `DocType` value is stored in the `docType` variable. You can use the xref:dataweave-selectors.adoc[metadata selector] (`^`) to extract the value of that variable with an expression like this one:
```

Comment:
This adds a link. We should use a full URL to the page in the source so that if it's shared in other places (outside our doc set), the link will work. 
```suggestion
During the reading phase, DataWeave parses the doctype declarations and stores the content as metadata of the root element. The `DocType` value is stored in the `docType` variable. You can use the https://docs.mulesoft.com/dataweave/latest/dataweave-selectors[metadata selector] (`^`) to extract the value of that variable with an expression like this one:
```

Reply:
that should also go in the source